extensions/aegis/src/component/otp/OtpList.tsx

```diff
@@ -1,3 +1,4 @@
+// OtpList.tsx
```

Comment:
Would you mind removing this useless header?

Reply:
Thanks, I’ve removed it.